packages/manager/apps/web/client/app/domain/zone/activate/translations/Messages_en_GB.json

```diff
@@ -5,7 +5,6 @@
   "web_domain_zone_activation_question": "Do you want to enable the minimum records in your DNS zone?",
   "web_domain_zone_activation_template_minimized": "Yes, I want to enable my DNS zone with minimum records.",
   "web_domain_zone_activation_template_basic": "No, but I want to enable my DNS zone.",
-  "web_domain_zone_activation_success": "The purchase order has been generated.",
```

Comment:
Removed in https://github.com/ovh/manager/commit/3641be1169e02af1d9faf12254c19573a3856194#diff-39b0aa64ac64fce99dfc0cd7616dad55L8